From 83dda2f132cd89e888e230450af8a35e23052b93 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Thu, 28 Mar 2024 13:24:49 +0800
Subject: Change native-comp-async-report-warnings-errors to silent

We have had numerous bug reports across packages about this bothering
users.
---
 lisp/emacs-lisp/comp.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lisp/emacs-lisp/comp.el b/lisp/emacs-lisp/comp.el
index cf8f5395a64..257e36e32db 100644
--- a/lisp/emacs-lisp/comp.el
+++ b/lisp/emacs-lisp/comp.el
@@ -144,7 +144,7 @@ native-comp-async-env-modifier-form
   :risky t
   :version "28.1")
 
-(defcustom native-comp-async-report-warnings-errors t
+(defcustom native-comp-async-report-warnings-errors 'silent
   "Whether to report warnings and errors from asynchronous native compilation.
 
 When native compilation happens asynchronously, it can produce
